components/job-orchestration/job_orchestration/executor/compress/compression_task.py

```diff
@@ -176,7 +246,7 @@ def run_clp(
     yaml.safe_dump(clp_metadata_db_connection_config, db_config_file)
     db_config_file.close()
 
-    # Get s3 config
+    # Get S3 config
```

Comment:
The docstring of this function should probably be updated since the source is no longer only FS.